app/ci_program.py

```diff
@@ -592,6 +592,8 @@ def parseArgs(self):
         elif i == '--clearHistory':
           app.history.clearUserHistory()
           self.quitNow()
+        elif i == '--eightColors':
+          numColors = 8
```

Comment:
Thanks for adding the --eightColors switch. Consider making this its own PR and we can land it straight away (and it would make this PR smaller, which is nice).

Reply:
Okay I've created a new PR, though I'm not really sure of a clean way to remove all relevant changes without manually doing it and having manual cleanup.

Reply:
I've just merged the other PR. I *think* doing a merge to head will remove the changes shown in this PR (that are in that PR).

Reply:
Nice!